hermes.py

```diff
@@ -0,0 +1,80 @@
+#! /usr/bin/env python3
+
+import sys
+import subprocess
+import click
+import gcd
+import time
+import requests
+import yaml
+
+
+@click.command()
+@click.option('--daemon/--no-daemon', default=False)
+@click.option('--debug/--no-debug', default=False)
+@click.option('--config', '-c', type=(str, str, str, int), help='filename url update_command seconds_between_updates', multiple=True)
+@click.option('--yamlfile', '-y', type=click.Path())
+
+def watch_config(config, daemon, debug, yamlfile):
+    if yamlfile:
+        with open(yamlfile) as yamlhandle:
+            config = yaml.load(yamlhandle)
+    seconds = 0
+    update_interval = gcd.gcd([update_intervals[3] for update_intervals in config])
+    if debug:
+        sys.stdout.write('DEBUG: Sleep Interval %d\n' % update_interval)
+    file_timestamps = {}
+
+    while True:
+        for config_file in config:
```

Comment:
Do you mean a config block?  Can you provide a sample of what the yaml file looks like?